src/lib/context/theme-variants.ts

```
@@ -0,0 +1,38 @@
+ * @returns The default theme variant string for the given experience type.
+ */
+export function getDefaultVariant(exp: GetDefaultVariantExperience): ThemeVariant {
+  switch (exp) {
+    case 'standard':
+      return standardVariants[0];
+    case 'corporate':
+    case 'both':
+      return corporateVariants[0];
+    default:
+      // This ensures type safety by exhaustively checking all cases
+      const _exhaustiveCheck: never = exp;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)